quickcache/quickcache_helper.py

```diff
@@ -76,6 +76,14 @@ def get_cached_value(self, *args, **kwargs):
         logger.debug(key)
         return self.cache.get(key, default=Ellipsis)
 
+    def set_cached_value(self, *args, **kwargs):
+        """
+        Sets the cached value
+        """
+        key = self.get_cache_key(*args, **kwargs)
+        logger.debug(key)
+        self.cache.set(key, default=Ellipsis)
```

Comment:
I think you want it to do something else here. Conceptually (in general) gets are `get(key)` and sets are `set(key, value)`. Here, the "key" is passed in as `*args, **kwargs`—this is so that you can call this function the same way you'd call the main function.

There are two ways we could set up the interface: pass `value` as the _first_ arg to this function, or have this function return a callable and pass `value` to that callable. For example, if you have

```
@quickcache(['self.id'])
def get_friends(self):
    ...
```

you call `self.get_friends.get_cached_value(self)` to get the value from the cache, `self.get_friends.clear(self)` to clear the cache. Our choices for the interface to set the cache are `self.get_friends.set_cached_value(value, self)` and `self.get_friends.set_cached_value(self)(value)`. Or `self.get_friends.set_cached_value(self).to(value)` or some such thing.

I'm currently leaning towards `self.get_friends.set_cached_value(self).to(value)`. You can achieve that by using

```
return namedtuple('Settable', ['to'])(lambda value: self.cache.set(key, value))
```